docs/sql-ref-syntax-ddl-create-view.md

```diff
@@ -19,4 +19,53 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+A view is a virtual table based on the result set of base query. The base query
+can involve joins, expressions, reordered columns, column aliases, and other SQL
+features that can make a query hard to understand or maintain.
+A view is purely a logical construct (an alias for a query) with no physical
+data behind it.
+
+### Syntax
+{% highlight sql %}
+CREATE [OR REPLACE] [[GLOBAL] TEMPORARY] VIEW [IF NOT EXISTS] [db_name.]view_name
+    [(column_name [COMMENT column_comment], ...) ]
+    create_view_clauses
+    AS SELECT ...;
+    
+    create_view_clauses (order insensitive):
+        [COMMENT view_comment]
+        [TBLPROPERTIES (property_name = property_value, ...)]
+{% endhighlight %}
+
+### Examples
+{% highlight sql %}
+-- Create a global temporary view `v1` if it does not exist.
+CREATE GLOBAL TEMPORARY VIEW IF NOT EXISTS v1 AS SELECT * FROM t1;
```

Comment:
@amanomer Can we make this example be a little real-world like ? Like a view that contains customers for state CA or sales for month of january .. IMHO select * from t1 probably wouldn't require a view , right ?

Reply:
You are definitely right. I am re-writing this. 
Thanks for your review.